Gemfile

```diff
@@ -31,7 +31,7 @@ gem "bundler",                          "~> 2.1", ">= 2.1.4", "!= 2.2.10", :requ
 gem "byebug",                                                :require => false
 gem "color",                            "~>1.8"
 gem "config",                           "~>2.2", ">=2.2.3",  :require => false
-gem "dalli",                            "=2.7.6",            :require => false
+gem "dalli",                            "~>2.7.8",           :require => false
```

Comment:
The dalli gem is up to 2.7.11 (https://rubygems.org/gems/dalli/versions).  Would it make sense to be more aggressive on the versions - something like `>= 2.7.8`

Reply:
We don't want >= cause we don't want to pull in major versions, but this should probably be "~> 2.7.11" once that it verified.

Reply:
Merged